lib/y2_module_guitest.pm

```diff
@@ -60,7 +62,8 @@ sub launch_yast2_module_x11 {
     diag 'assuming root-auth-dialog, typing password';
     type_password;
     send_key 'ret';
-    assert_screen $args{target_match}, $args{match_timeout};
+    wait_still_screen(timeout => 300);
```

Comment:
Does this needed? If the timeout is not enough, I think we can increase the timeout value when run the launch_yast2_module_x11.

Reply:
> Does this needed? If the timeout is not enough, I think we can increase the timeout value when run the launch_yast2_module_x11.

Sometimes it'll freeze here till timeout https://openqa.suse.de/tests/13239257#step/yast2_instserver/10, and the workaround helps.